src/Controller/Rest/ApiController.php

```diff
@@ -17,47 +14,44 @@ class ApiController extends FOSRestController
 {
 
     /**
-    * Initialize calendar
-    * @Rest\Get("/calendar")
-    */
-    public function getCalendar()
+     * Initialize calendar
+     * @Rest\Get("/calendar")
+     * @return View
+     */
+    public function getCalendar(): View
     {
-        $sessionRepo = $this->getDoctrine()->getRepository(Session::class);
+        $sessionRepo = $this->getDoctrine()->getRepository(Session::class)->findBy(['status' => 'free']);
```

Comment:
naming'as labai missleading `sessionRepo` gali buti sessiju repositorija (`$this->getDoctrine()->getRepository(Session::class)`), bet cia tu gauni rezultatus, tai - $sessions